### DISCUSSION TITLE
Use arrow key to move cursor within one input line

### DISCUSSION THREAD

**Discussion by zuckerruebe** [wxMaxima-developers/wxmaxima#1849] [Q&A]
Hi there

New to (wx)Maxima, but liking it a lot!

I somehow can't manage to move the cursor within an input line using the arrow keys on the keyboard: I always have to point and click using the mouse...

Is it something weird about my setup that's causing that or are other people experiencing this as well?

Cheers,
Damian

**Comment by zmth** [comment]
yea i know. This is a most horrible thing about maxima that has never been fixed or done all these years ! such a royal pain in the ass ! In reading text files, .pdf, most any other files , even just plain dos output in fortran and for example this answer or comment box ,  one expects to use the up/down arrow keys to advance or scroll back one text line at a time. But not maxima output ! And i hate having to use that damn mouse !! to go over and grab the bar at the right or wherever . What a royal pain ! The navigation in maxima input/output is so so bad. Why that has not been  fixed or provided i do not know. It makes navigation such a pain in maxima that one should not have to put up with that kind of crap. Of course one input line is generally a whole program of many text lines and most often so is the output. They act as if one never does any programming and only puts things which are at most one or maybe 2 text lines and also for the output. Doing things to fit that format is so unnatural as so so inefficient and wasteful and ofcourse one is going to want to look at for example every line of a matrix etc. which could be several pages. Yea there is stringout where one can output to a file but that is a pain and inefficient also. One wants to be able to interact with maxima input/output text lines right then and there directly and would certainly expect to be able to do it in a reasonable way just as most any other compiler or whatever. If nothing else put this in a feature request ! It's something that should have been provided from the beginning of maxima from macsyma. so put in a feature request as not having that what anyone should expect makes dealing with maxima so so inefficient !!
